pkgs/top-level/perl-packages.nix

```diff
@@ -4657,6 +4657,20 @@ let
     propagatedBuildInputs = [ Curses TermReadKey ];
   };
 
+  CursesUIGrid = buildPerlPackage {
+    pname = "Curses-UI-Grid";
+    version = "0.15";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/A/AD/ADRIANWIT/Curses-UI-Grid-0.15.tar.gz";
+      sha256 = "0820ca4a9fb949ba8faf97af574018baeffb694e980c5087bb6522aa70b9dbec";
+    };
+    propagatedBuildInputs = with perlPackages; [ CursesUI TestPod TestPodCoverage ];
```

Comment:
`with perlPackages;` is not needed here

Reply:
You're right, i forgot to change it from when i test it :D
Gonna fix this